appnav/src/main/kotlin/io/element/android/appnav/NotLoggedInFlowNode.kt

```diff
@@ -83,6 +88,10 @@ class NotLoggedInFlowNode @AssistedInject constructor(
                     override fun onSignIn() {
                         backstack.push(NavTarget.LoginFlow(isAccountCreation = false))
                     }
+
+                    override fun onOpenDeveloperSettings() {
+                        backstack.push(NavTarget.ConfigureTracing)
```

Comment:
Maybe in the near future we should change this to opening the developer settings, so we can change feature flags, i.e.?

Reply:
Yes, you're right, and I though about it. But currently `DeveloperSettingsNode` needs `SessionScope`, so there is more work to do first.